docs/topics/i18n/translation.txt

```diff
@@ -825,6 +825,10 @@ Each ``RequestContext`` has access to three translation-specific variables:
   right-to-left language, e.g.: Hebrew, Arabic. If False it's a
   left-to-right language, e.g.: English, French, German etc.
 
+.. versionchanged:: 1.8
+
+  The `i18n` context processor is not enabled by default for new projects.
```

Comment:
I would use as I wouldn't expect things like i18n and tz to pass the spell check (but maybe they do)